process.md

```diff
@@ -143,3 +143,9 @@ Thank you,
 The MicroProfile core team
 
 ---
+
+## Steps after a proposal has been accepted
+
+* Announce in the review mailing thread that the proposal has been accepted and that repositories are being created
+* Raise a bug for Eclipse to create a Github repsitory or ask for migration of an existing repository (see the [Eclipse Handbook](https://www.eclipse.org/projects/handbook/#resources-github))
+** When repositories are created, announce in a new thread that the proposal can be worked on in the new repository
```

Comment:
s/proposal can be worked on/proposal can be implemented/ ?

As I understand the 'accepted' state, it means that the proposal needs to be fully fleshed out and "cast in stone" (except or accept with revisions).

I was confused about the 'accept' state, as in the original discussions it meant that the proposal(-pull request) is deemed useful for MP, but that the final work and touches on it would be done in the 'accepted' state.